crates/oxide/src/glob.rs

```diff
@@ -136,6 +139,14 @@ fn get_fast_patterns(base_path: &Path, patterns: &Vec<String>) -> Vec<(PathBuf,
     optimized_patterns
 }
 
+pub fn path_matches_globs(path: &Path, globs: &[GlobEntry]) -> bool {
+    let path = format!("{}", path.display());
```

Comment:
The intention here is just to convert the `path` to string, right? Perhaps we can use `Path#to_string_lossy` instead to make it clearer and avoid the intermediate display representation?